[CI] Rename the inductor test config names for dynamic shapes tests

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #113668
* #113559
* #113446
* #113575
* __->__ #113574

Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @chenyang78 @aakhundov @kadeng

desertfire committed:
[CI] Rename the inductor test config names for dynamic shapes tests

Summary: To make the naming consistent with tests in inductor-periodic and simply update_expected.py

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

malfet reviewed (approved):
Please note that those kind of renames will result in CI failures until folks will rebase on the latest version (because workflows definition are always rebased against latest trunk, but CSV renames are not)

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire commented:
> Please note that those kind of renames will result in CI failures until folks will rebase on the latest version (because workflows definition are always rebased against latest trunk, but CSV renames are not)

I will land this stack of PRs in one merge and notify people to rebase, because there is another similar renaming for cpu_inductor tests.

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]

desertfire committed:
Update on "[CI] Rename the inductor test config names for dynamic shapes tests"


Summary: To make the naming consistent with tests in inductor-periodic and simplify update_expected.py

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx chenyang78 aakhundov kadeng

[ghstack-poisoned]
